README.md

```diff
@@ -3,34 +3,65 @@
 > This is a [Lens][lens] extension for [Starboard][starboard] which provides visibility into
 > vulnerability assessment reports for Kubernetes workloads stored as [custom security resources][starboard-crds].
 
+[![GitHub Release][release-img]][release]
 [![GitHub Build Actions][build-action-img]][actions]
 [![License][license-img]][license]
 
 [![Screenshot](docs/images/starboard-lens-extension-video.png)](https://youtu.be/X-bhVwmp2l4)
 
 ## Installation
 
+This guide shows how to install the extension either from source or from pre-built binary releases.
+
 ### Prerequisites
 
-* Lens >= 4.0.0-beta.4 should first be installed. Download a pre-built package from the [releases][lens-releases] page.
-  Alternatively, on macOS installing the latest version is as simple as `brew cask install lens`. For installation
-  instructions on other operating systems and packages managers, see [Lens Installation][lens-installation].
+* Lens >= [4.0.0-rc1.4][min-lens] should first be [installed][lens-installation].
+* Follow Starboard [Getting Started][starboard-getting-started] guide to understand how Starboard is
+  initialized and how it generates security reports, which are then stored as custom resources.
+
+### From the Binary Releases
+
+Every [release][release] of Lens extension for Starboard provides the tarball-file that can be manually
+downloaded and installed. Alternatively, you can copy the download URL of the release artifact to paste it in the
+**Manage Lens Extensions** page.
+
+1. Download your desired version from GitHub [release][release] page or GitHub
+   [package registry](https://github.com/aquasecurity/starboard-lens-extension/packages/520430). Alternatively,
```

Comment:
Could we make this link into a variable too?

Reply:
Sure. I've changed to reference style link